### ISSUE TITLE
abacus doesn't load due to parsing error and error messages not being seen as strings

### ISSUE THREAD

**Issue by MrKevinOConnell** [acm-mu/abacus#175]
## Description

- upon setting up abacus in dev this error shows

- in addition to this, abacus won't load due to error messages not being seen as strings

```
./node_modules/@toast-ui/react-chart/dist/toastui-react-chart.js 59:13
Module parse failed: Unexpected token (59:13)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|   function s(t) {
|     return class extends a().Component {
>       rootEl = a().createRef();
|       chartInst = null;
|
```

### Steps to reproduce

- run npm install in both `frontend` and `backend`
- run npm run dev in `backend`
- run npm start in `frontend`

### Expected result

- abacus loads successfully

### Actual result

- the above error shows

### Evidence

<img width="500" alt="Screen Shot 2021-09-09 at 5 22 02 PM" src="https://user-images.githubusercontent.com/17347501/132770288-c6a6aef1-1b69-4f61-b958-a3f0b60230bb.png">

**Comment by Jl5her** [comment]
@MrKevinOConnell Is this issue resolved as a result of #176?

**Comment by MrKevinOConnell** [comment]
> @MrKevinOConnell Is this issue resolved as a result of #176?

@Jl5her yes!
